campusmap/src/info/Info/Info.tsx

```diff
@@ -0,0 +1,73 @@
+ * Individual Info Result component rendered in a carousel.
+ */
+
+ const Info = ({ location }: Props): ReactElement => {
+    return (
+        <body>
+            <Jumbotron>
+                <Card className="mb-4">
+                    <Row className="p-4">
+                        <Col className="col-2">
+                            <Card.Img src={"..."} style={{ maxWidth: '100px' }} alt={location.properties.name} />
+                        </Col>
+                        <Col className="col-10">
+                            <Card.Body>
+                            <Card.Title>{location.properties.name}</Card.Title>
+                            <Card.Text>{location.properties.description}</Card.Text>
+                            {location.properties.nick 
+                                && <Card.Subtitle>Also known as:&nbsp;{location.properties.nick}</Card.Subtitle>}
+                            </Card.Body>
+                        </Col>
+                        <Button variant="outline-danger" href="`/info?loc=${location._id}`">Show on Map</Button>
+                    </Row>
+                </Card>
+                <Carousel>
+                    <Carousel.Item>
+                        <img 
+                            src="http://www.alpha6294.com/History/ThetaXionCampus/DarrinCommunicationsCenter/Building/DarrinCommunicationsCenter.jpg"
+                            className="d-block w-100" 
+                            alt="..."
+                        />
+                    </Carousel.Item>
+                    <Carousel.Item>
+                        <img 
+                            src="https://www.bing.com/th?id=OIP.npiBC_gFZvex2KDN16YTZAHaFj&pid=Api&rs=1&p=0"
+                            className="d-block w-100"                   
+                            alt="..."
+                        />
+                    </Carousel.Item>
+                </Carousel>
+            </Jumbotron>
+            <Container>
+                <h2>Comments:</h2>
+                <h5>Post a Comment:</h5>
+                <Form>
+                    <Form.Group controlId="commentTitle">
+                        <Form.Label>Comment Title</Form.Label>
+                        <Form.Control type="text" placeholder="Title" />
+                        <Button as="input" type="submit" value="Post Comment" />{' '}
+                    </Form.Group>
+                    <Form.Group controlId="commentBody">
+                        <Form.Control as="textarea" rows="3" placeholder="Type your comment here..." />
+                    </Form.Group>
+                    <Form.Group controlId="imageLink">
+                        <Form.Label>Add an Image:</Form.Label>
+                        <Form.Control type="url" placeholder="Enter image link" />
+                        <Button as="input" type="submit" value="Submit Image" />{' '}
+                    </Form.Group>
+                </Form>
+            </Container>
+        </body>
+    );
+ };
```

Comment:
there should be a newline at the end of the file and this function should be exported